[Flyout System] Service to open flyouts within the Flyout System

tsullivan opened the pull request:
~~Depends on https://github.com/elastic/eui/pull/8982~~
~~Depends on https://github.com/elastic/eui/pull/9202~~
~~Depends on https://github.com/elastic/kibana/pull/245515~~

This PR enhances the core UX offerings in Kibana with `core.overlays.openSystemFlyout`.

### `core.overlays.openSystemFlyout`
Opens a system flyout that integrates with the EUI Flyout Manager. Using a mount point would break the context propogation of the EUI Flyout Manager, so this method accepts React elements directly rather than `toMountPoint`.
```typescript
import React, { useRef } from 'react';
import { 
  EuiFlyoutBody, 
  EuiFlyoutFooter,
  EuiText,
  EuiButton,
  EuiButtonEmpty,
  EuiFlexGroup,
  EuiFlexItem 
} from '@elastic/eui';
import type { OverlayRef } from '@kbn/core-mount-utils-browser';

// Create a component or function that opens the system flyout
const openMySystemFlyout = (overlays) => {
  const flyoutRef = useRef<OverlayRef | null>(null);

  const handleClose = () => {
    if (flyoutRef.current) {
      flyoutRef.current.close();
    }
  };

  const FlyoutContent = () => (
    <>
      <EuiFlyoutBody>
        <EuiText>
          <p>This is a system flyout that integrates with EUI Flyout Manager.</p>
          <p>The header is automatically created from the title option.</p>
        </EuiText>
      </EuiFlyoutBody>
      <EuiFlyoutFooter>
        <EuiFlexGroup justifyContent="spaceBetween">
          <EuiFlexItem grow={false}>
            <EuiButtonEmpty onClick={handleClose}>
              Cancel
            </EuiButtonEmpty>
          </EuiFlexItem>
          <EuiFlexItem grow={false}>
            <EuiButton onClick={() => console.log('Save')} fill>
              Save
            </EuiButton>
          </EuiFlexItem>
        </EuiFlexGroup>
      </EuiFlyoutFooter>
    </>
  );

  flyoutRef.current = overlays.openSystemFlyout(<FlyoutContent />, {
    title: 'My System Flyout',
    type: 'overlay',
    size: 'm',
    maxWidth: 600,
    ownFocus: false,
    onClose: () => {
      console.log('System flyout closed');
      flyoutRef.current = null;
    },
    onActive: () => {
      console.log('System flyout became active');
    },
  });

  return flyoutRef.current;
};

// Open the flyout
const flyoutRef = openMySystemFlyout(overlays);

// Close the flyout programmatically from outside
flyoutRef.close();
```

The developer README for the Flyout System is [here](https://github.com/elastic/eui/blob/feat/flyout-system/packages/eui/src/components/flyout/README.md). 

### Example plugin

Along with the new service and documentation changes, a new example plugin for developers is provided that shows different integration patterns for the EUI flyout system.

**Screenshot of example plugin:**
<img width="1327" height="836" alt="image" src="https://github.com/user-attachments/assets/9664f0cb-f793-4a8c-8a96-c46d069756dd" />

tsullivan committed:
Example of Flyout System in Kibana Example 

Take 3

tsullivan committed:
Example of Flyout System in Kibana Example 

Take 3

tsullivan committed:
Plugin rename

tsullivan committed:
Plugin rename

tsullivan committed:
Expand Examples

tsullivan committed:
Update example plugin setup code

tsullivan committed:
Update example plugin setup code

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

tsullivan committed:
update child flyout state when main flyout is closed

tsullivan committed:
Flyout props for demo

tsullivan committed:
Fix warning of duplicate flyout IDs

tsullivan committed:
Add callbacks for state synchronization

tsullivan committed:
Add example tabs and core.overlays.openFlyout

tsullivan committed:
update openFlyout example

tsullivan committed:
Fix example page router

tsullivan committed:
refactor: enhance flyout system examples with shared utilities and consistent UI

- Add shared utilities in utils/ directory for flyout components
- Create FlyoutTypeSwitch component for consistent push/overlay toggles
- Add description list utilities for consistent flyout information display
- Update all flyout examples to use shared utilities and consistent styling
- Add EuiCode styling to flyout titles for better visual distinction
- Implement onActive/onClose callbacks for state synchronization
- Add URL-based routing for tab persistence on page refresh
- Standardize session naming: A-G (with isOpen), J-L (without isOpen), X-Z (overlays)
- Remove global flyout type controls in favor of per-session controls
- Improve code organization and reduce duplication across examples

tsullivan committed:
remove "flyout with isOpen" example

tsullivan committed:
fix references

tsullivan committed:
Add Non-Session Flyout to example

tsullivan committed:
Install new build of EUI flyout system feature branch, no.29

tsullivan committed:
fix ownFocus for example

tsullivan committed:
update per breaking api changes

tsullivan committed:
New openSystemFlyout method in core.overlays

new openSystemFlyout

tsullivan committed:
Prevent nav flyouts from becoming child in session

tsullivan commented:
/ci

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/353771)
* Commit: 9c02661dd5a5ca20084cb0425d2b18f51f02af16

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/353771#019a0d0e-b1ef-4217-9ea7-4abdcdf087ee)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/353771","number":353771,"commit":"9c02661dd5a5ca20084cb0425d2b18f51f02af16"}],"number":353771}
buildkite-pr-comment-->

tsullivan committed:
Fix ref typings in use_flyout_a11y

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/notice'

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/generate codeowners'

tsullivan committed:
fix types

quick fix for type check failure

fix types iii

tsullivan committed:
Install custom EUI build of feature branch no.30 for type check testing

tsullivan commented:
/ci

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/354232)
* Commit: 1b44c8803f7519d2d72c240b9f13d3d42d37f577

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/354232#019a122a-7e39-4653-9ad1-10059e057116)

### History
* :broken_heart: [Build #353800](https://buildkite.com/elastic/kibana-pull-request/builds/353800) failed d05cb767b06427a504e41fd2865a9cfdfcb153ad
* :broken_heart: [Build #353771](https://buildkite.com/elastic/kibana-pull-request/builds/353771) failed 9c02661dd5a5ca20084cb0425d2b18f51f02af16

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/354232","number":354232,"commit":"1b44c8803f7519d2d72c240b9f13d3d42d37f577"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/353800","number":353800,"commit":"d05cb767b06427a504e41fd2865a9cfdfcb153ad"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/353771","number":353771,"commit":"9c02661dd5a5ca20084cb0425d2b18f51f02af16"}],"number":354232}
buildkite-pr-comment-->

tsullivan committed:
Install custom EUI build of feature branch no.30

tsullivan committed:
EUI Flyout System Build no.31

tsullivan committed:
Refine examples

akowalska622 committed:
adjust use flyout a11y types

akowalska622 committed:
doc viewer flyout refactor

akowalska622 committed:
add action.inspect to get_columns

akowalska622 committed:
first attempt of background search session flyout

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
add ability openSystemFlyout to open a child flyout

tsullivan committed:
Example app polish

tkajtoch committed:
fix: update EuiFlyout `ref` usage to not use string refs

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Install custom EUI v108 build no.32 for testing

tsullivan committed:
Flyout System integration for SharedUX example code

tsullivan committed:
Flyout System integration for SharedUX content management

tsullivan committed:
Flyout System integration for Connection Details

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

tsullivan committed:
Test: eui-theme-common ignore dependency ownership logic

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/357342)
* Commit: fd4eeb5afca6f23d3b44f2e298f0f1d1178f28f3

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/357342#019a3bb2-271b-4f9a-ad60-e0d9b1803595)

### History
* :broken_heart: [Build #357326](https://buildkite.com/elastic/kibana-pull-request/builds/357326) failed 9b41fa95c59f004dbf175646bb231a167c8f5c13
* :broken_heart: [Build #354232](https://buildkite.com/elastic/kibana-pull-request/builds/354232) failed 1b44c8803f7519d2d72c240b9f13d3d42d37f577
* :broken_heart: [Build #353800](https://buildkite.com/elastic/kibana-pull-request/builds/353800) failed d05cb767b06427a504e41fd2865a9cfdfcb153ad
* :broken_heart: [Build #353771](https://buildkite.com/elastic/kibana-pull-request/builds/353771) failed 9c02661dd5a5ca20084cb0425d2b18f51f02af16

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357342","number":357342,"commit":"fd4eeb5afca6f23d3b44f2e298f0f1d1178f28f3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357326","number":357326,"commit":"9b41fa95c59f004dbf175646bb231a167c8f5c13"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/354232","number":354232,"commit":"1b44c8803f7519d2d72c240b9f13d3d42d37f577"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/353800","number":353800,"commit":"d05cb767b06427a504e41fd2865a9cfdfcb153ad"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/353771","number":353771,"commit":"9c02661dd5a5ca20084cb0425d2b18f51f02af16"}],"number":357342}
buildkite-pr-comment-->

tsullivan committed:
Fix types - WIP

tsullivan committed:
fix types: data plugin

tsullivan committed:
Type fixes for openSystemFlyout service

akowalska622 committed:
Revert "adjust use flyout a11y types"

This reverts commit cb81323e7a9a7eaedf5df4c8d13bfdfaa41aabce.

akowalska622 committed:
Revert "doc viewer flyout refactor"

This reverts commit 0daf127012da64abd6135f160a0e811c13ffe9a8.

akowalska622 committed:
Revert "add action.inspect to get_columns"

This reverts commit bc567ffc156c6675187bc370345e42ef7656663d.

akowalska622 committed:
Revert "first attempt of background search session flyout"

This reverts commit 1048e61b9d560dd217d32b474dc38b87077319aa.

akowalska622 committed:
revert get_columns.tsx

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
fix EuiFlyoutResizable ref types

tsullivan committed:
Flyout System for Newsfeed

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/357884)
* Commit: c834a122a624fa921e150f5a27ad8ea07672f517

### Failed CI Steps
* [Jest Tests #16](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b9f-1f4d-4485-8355-3b246f1b8f60)
* [Scout Test Run Builder](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e)
* [Scout Test Run Builder](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b9f-1f4d-4485-8355-3b246f1b8f60) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b9f-1f4d-4485-8355-3b246f1b8f60/artifacts/019a4bb5-152b-432a-89d6-4d201d5d8836) Jest Tests #<span></span>16 / #<span></span>start() returns `Context` component
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b7c-873f-410d-8e53-7faa1bd8ad4e/artifacts/019a4ba6-889b-4193-928d-658a12124130) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiCheckBox - checkbox
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4ba6-9d58-4fb6-8a18-bf92ad8011e8/artifacts/019a4bb2-ad46-42e8-ac13-e034c9f5705e) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiCheckBox - checkbox
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b7c-873f-410d-8e53-7faa1bd8ad4e/artifacts/019a4ba6-889b-4a3c-a72e-24257533da45) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with multiple selections (pills)
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4ba6-9d58-4fb6-8a18-bf92ad8011e8/artifacts/019a4bb2-ad46-4711-b837-4581631a77fa) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with multiple selections (pills)
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b7c-873f-410d-8e53-7faa1bd8ad4e/artifacts/019a4ba6-889a-43d4-9af6-0e275878e9a1) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with the single selection
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4ba6-9d58-4fb6-8a18-bf92ad8011e8/artifacts/019a4bb2-ad44-412e-acbb-0d5dbdc35a7d) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with the single selection
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b7c-873f-410d-8e53-7faa1bd8ad4e/artifacts/019a4ba6-8899-4aa2-9308-fddf27229f8c) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiDataGrid - data grid, run
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4ba6-9d58-4fb6-8a18-bf92ad8011e8/artifacts/019a4bb2-ad44-496e-a0c9-0f8c2f645783) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiDataGrid - data grid, run
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b7c-873f-410d-8e53-7faa1bd8ad4e/artifacts/019a4ba6-889b-4ca8-a3b1-ce63da52fb4e) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiSelectable - selectable with search field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4ba6-9d58-4fb6-8a18-bf92ad8011e8/artifacts/019a4bb2-ad45-477c-8ca2-2af91e8020ed) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiSelectable - selectable with search field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4b7c-873f-410d-8e53-7faa1bd8ad4e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4b7c-873f-410d-8e53-7faa1bd8ad4e/artifacts/019a4ba6-889a-4d93-945a-4ddf5d843ee6) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiToast - toast
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/357884#019a4ba6-9d58-4fb6-8a18-bf92ad8011e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/357884/jobs/019a4ba6-9d58-4fb6-8a18-bf92ad8011e8/artifacts/019a4bb2-ad44-467e-84d5-a5b1beef3ba2) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiToast - toast

### History
* :broken_heart: [Build #357389](https://buildkite.com/elastic/kibana-pull-request/builds/357389) failed 5179dfa96ed18bafbcc3c8e99f3ae5445fd2b753
* :broken_heart: [Build #357368](https://buildkite.com/elastic/kibana-pull-request/builds/357368) failed 303bf88413a5d85ba6ab87b873a93aa8ccfbad65
* :broken_heart: [Build #357342](https://buildkite.com/elastic/kibana-pull-request/builds/357342) failed fd4eeb5afca6f23d3b44f2e298f0f1d1178f28f3
* :broken_heart: [Build #357326](https://buildkite.com/elastic/kibana-pull-request/builds/357326) failed 9b41fa95c59f004dbf175646bb231a167c8f5c13
* :broken_heart: [Build #354232](https://buildkite.com/elastic/kibana-pull-request/builds/354232) failed 1b44c8803f7519d2d72c240b9f13d3d42d37f577

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357884","number":357884,"commit":"c834a122a624fa921e150f5a27ad8ea07672f517"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357389","number":357389,"commit":"5179dfa96ed18bafbcc3c8e99f3ae5445fd2b753"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357368","number":357368,"commit":"303bf88413a5d85ba6ab87b873a93aa8ccfbad65"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357342","number":357342,"commit":"fd4eeb5afca6f23d3b44f2e298f0f1d1178f28f3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357326","number":357326,"commit":"9b41fa95c59f004dbf175646bb231a167c8f5c13"}],"number":357884}
buildkite-pr-comment-->

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Have main flyout contents be scrollable

tsullivan committed:
Overlays example: Clean up child flyout state management from FlyoutSession

tsullivan committed:
Consistency

tsullivan committed:
Install custom EUI v108 build no.33 for testing

tsullivan committed:
Install custom EUI v108 build no.34 for testing

tsullivan committed:
Install custom EUI build of feature branch no.35

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/358358)
* Commit: 33acdb3037521b462720d864b70252463289c56a

### Failed CI Steps
* [Jest Tests #12](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f8-4a75-bded-001ac238b72f)
* [Jest Tests #13](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f9-4f99-a2e4-8bb0e4cbdaff)
* [Scout Test Run Builder](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09)
* [Scout Test Run Builder](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f9-4f99-a2e4-8bb0e4cbdaff) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5143-29f9-4f99-a2e4-8bb0e4cbdaff/artifacts/019a5155-c463-400e-b626-a566ef41f500) Jest Tests #<span></span>13 / #<span></span>start() returns `Context` component
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f8-4a75-bded-001ac238b72f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5143-29f8-4a75-bded-001ac238b72f/artifacts/019a5159-bdc3-4e6d-af89-5a430c75e52b) Jest Tests #<span></span>12 / <IconButtonGroup /> is rendered
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f8-4a75-bded-001ac238b72f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5143-29f8-4a75-bded-001ac238b72f/artifacts/019a5159-bdc1-491c-a14f-502f93cb6902) Jest Tests #<span></span>12 / <SimplePrivilegeForm> renders without crashing
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f8-4a75-bded-001ac238b72f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5143-29f8-4a75-bded-001ac238b72f/artifacts/019a5159-bdc0-44f3-9c2f-899c32f4e5b4) Jest Tests #<span></span>12 / it renders without blowing up
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5143-29f8-4a75-bded-001ac238b72f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5143-29f8-4a75-bded-001ac238b72f/artifacts/019a5159-bdc1-4eec-a9db-9f36b25281a3) Jest Tests #<span></span>12 / LoginForm renders as expected
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5120-8261-4b1f-a38a-b7fafd423c09/artifacts/019a514a-16a8-4e47-957b-6c60a8a86aa6) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiCheckBox - checkbox
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a514a-2a3f-45a6-8631-0b091bb4da84/artifacts/019a5156-3acb-4ceb-8337-254e12584d7e) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiCheckBox - checkbox
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5120-8261-4b1f-a38a-b7fafd423c09/artifacts/019a514a-16a8-4bcc-9d67-90338ab35162) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with multiple selections (pills)
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a514a-2a3f-45a6-8631-0b091bb4da84/artifacts/019a5156-3acb-4aa1-a821-16f49b518233) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with multiple selections (pills)
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5120-8261-4b1f-a38a-b7fafd423c09/artifacts/019a514a-16a7-4d83-ba9c-c2e5b2b00232) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with the single selection
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a514a-2a3f-45a6-8631-0b091bb4da84/artifacts/019a5156-3aca-45ab-a96c-ca03dc604729) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiComboBox - with the single selection
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5120-8261-4b1f-a38a-b7fafd423c09/artifacts/019a514a-16a7-426f-b91f-ea05e3a538f6) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiDataGrid - data grid, run
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a514a-2a3f-45a6-8631-0b091bb4da84/artifacts/019a5156-3aca-4197-a71e-652e2c08ecca) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiDataGrid - data grid, run
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5120-8261-4b1f-a38a-b7fafd423c09/artifacts/019a514a-16a8-4081-a4b1-398446973d5a) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiSelectable - selectable with search field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a514a-2a3f-45a6-8631-0b091bb4da84/artifacts/019a5156-3acb-4720-9253-28cec8b75d5b) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiSelectable - selectable with search field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a5120-8261-4b1f-a38a-b7fafd423c09) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a5120-8261-4b1f-a38a-b7fafd423c09/artifacts/019a514a-16a7-4354-bcf1-c1b3b8b524de) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiToast - toast
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358358#019a514a-2a3f-45a6-8631-0b091bb4da84) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358358/jobs/019a514a-2a3f-45a6-8631-0b091bb4da84/artifacts/019a5156-3aca-44de-a6b2-4b48373abc8b) Scout Test Run Builder / serverless-security - EUI testing wrapper: EuiToast - toast

### History
* :broken_heart: [Build #357884](https://buildkite.com/elastic/kibana-pull-request/builds/357884) failed c834a122a624fa921e150f5a27ad8ea07672f517
* :broken_heart: [Build #357389](https://buildkite.com/elastic/kibana-pull-request/builds/357389) failed 5179dfa96ed18bafbcc3c8e99f3ae5445fd2b753
* :broken_heart: [Build #357368](https://buildkite.com/elastic/kibana-pull-request/builds/357368) failed 303bf88413a5d85ba6ab87b873a93aa8ccfbad65
* :broken_heart: [Build #357342](https://buildkite.com/elastic/kibana-pull-request/builds/357342) failed fd4eeb5afca6f23d3b44f2e298f0f1d1178f28f3
* :broken_heart: [Build #357326](https://buildkite.com/elastic/kibana-pull-request/builds/357326) failed 9b41fa95c59f004dbf175646bb231a167c8f5c13

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/358358","number":358358,"commit":"33acdb3037521b462720d864b70252463289c56a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357884","number":357884,"commit":"c834a122a624fa921e150f5a27ad8ea07672f517"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357389","number":357389,"commit":"5179dfa96ed18bafbcc3c8e99f3ae5445fd2b753"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357368","number":357368,"commit":"303bf88413a5d85ba6ab87b873a93aa8ccfbad65"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/357342","number":357342,"commit":"fd4eeb5afca6f23d3b44f2e298f0f1d1178f28f3"}],"number":358358}
buildkite-pr-comment-->

tsullivan committed:
Install custom EUI build of feature branch no.36

tsullivan committed:
Update some test snapshots

tsullivan committed:
Update examples to show Child-to-Child navigation

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Install custom EUI v109 build no.37 for testing

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Install custom EUI build of feature branch no.38

weronikaolejniczak committed:
chore(eui): use noItemsMessage prop in EuiInMemoryTable

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Install custom EUI build of flyout system feature branch no.39

Commit:
Update test to reflect role change of search input from "searchbox" to "combobox" in DataView list component

tsullivan committed:
Update child flyouts to use `session=“inherit”`

tsullivan committed:
revert ts-ignore

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Remove SharedUX test integrations

tsullivan committed:
Install custom EUI build of flyout system feature branch no.41

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

tsullivan committed:
Add euiFlyoutManaged.defaultTitle to getEuiContextMapping

tkajtoch committed:
chore: update i18n mapping

tkajtoch committed:
test: update snapshots in src/platform/packages/shared/shared-ux/page/solution_nav/jest.config.js

tkajtoch committed:
test: update snapshots in src/platform/plugins/private/vis_types/vislib/jest.config.js

tkajtoch committed:
test: update snapshots in x-pack/platform/plugins/shared/security/jest.config.js

mgadewoll committed:
chore: update eui packages

mgadewoll committed:
refactor: update message prop

mgadewoll committed:
refactor: remove obsolete color prop

mgadewoll committed:
refactor: check for available query arg

- EUI updated the type and the query might be null for text searches

mgadewoll committed:
test: use noItemsMessage prop

mgadewoll committed:
test: update snapshots

mgadewoll committed:
Merge branch 'main' into eui/v110.0.0

mgadewoll committed:
Merge branch 'main' into eui/v110.0.0

tkajtoch committed:
style: remove unnecessary `z-index` overrides

tkajtoch committed:
style(ActionLogButton): remove unnecessary `z-index` override

tsullivan committed:
Merge branch 'eui/v110.0.0' into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Install custom EUI build of flyout system feature branch no.42

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/369153)
* Commit: f0112582a3b8c20f0131d56a9c34a88f58d38afa

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/369153#019aea71-5323-418d-ac3c-ed61b690570e)
* [Checks](https://buildkite.com/elastic/kibana-pull-request/builds/369153#019aea71-5324-4356-859e-987ac073e1a7)

### History
* :broken_heart: [Build #366100](https://buildkite.com/elastic/kibana-pull-request/builds/366100) failed f681e3f5adaafc792b2d84f57369a9ed800e49ed
* :broken_heart: [Build #365926](https://buildkite.com/elastic/kibana-pull-request/builds/365926) failed fa17e7f9cfcebb019c7f95b8b037119d0ff73f58
* :broken_heart: [Build #360594](https://buildkite.com/elastic/kibana-pull-request/builds/360594) failed 255eb1ed7de3f40d6162881168cea705830ce314
* :broken_heart: [Build #358747](https://buildkite.com/elastic/kibana-pull-request/builds/358747) failed 22d602084e6371fd883e41df8bd6a4760091fd61
* :broken_heart: [Build #358358](https://buildkite.com/elastic/kibana-pull-request/builds/358358) failed 33acdb3037521b462720d864b70252463289c56a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/369153","number":369153,"commit":"f0112582a3b8c20f0131d56a9c34a88f58d38afa"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/366100","number":366100,"commit":"f681e3f5adaafc792b2d84f57369a9ed800e49ed"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/365926","number":365926,"commit":"fa17e7f9cfcebb019c7f95b8b037119d0ff73f58"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360594","number":360594,"commit":"255eb1ed7de3f40d6162881168cea705830ce314"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/358747","number":358747,"commit":"22d602084e6371fd883e41df8bd6a4760091fd61"}],"number":369153}
buildkite-pr-comment-->

kibanamachine committed:
Changes from node scripts/capture_oas_snapshot   --include-path /api/status   --include-path /api/alerting/rule/   --include-path /api/alerting/rules   --include-path /api/actions   --include-path /api/security/role   --include-path /api/spaces   --include-path /api/streams   --include-path /api/fleet   --include-path /api/saved_objects/_import   --include-path /api/saved_objects/_export   --include-path /api/maintenance_window   --include-path /api/agent_builder --update

kibanamachine committed:
TO FIX: Run node 'scripts/yarn_deduplicate && yarn kbn bootstrap' locally, or add an exception to src/dev/yarn_deduplicate/index.ts and then commit the changes and push to your branch

tsullivan committed:
unit tests for system flyout service

tsullivan committed:
Merge branch 'kbn-flyout-system/example-iii' of github.com:tsullivan/kibana into kbn-flyout-system/example-iii

tkajtoch committed:
chore: upgrade `@elastic/eui` to v111.0.0

tkajtoch committed:
chore: update styled-components mapping

kibanamachine committed:
TO FIX: Run node 'scripts/yarn_deduplicate && yarn kbn bootstrap' locally, or add an exception to src/dev/yarn_deduplicate/index.ts and then commit the changes and push to your branch

elasticmachine committed:
Merge branch 'main' into feat/eui-v111.0.0

tkajtoch committed:
test: fix runtime field input selector to not target the overlay mask's class name

tkajtoch committed:
fix: remove remaining `z-index` overrides related to flyouts

tkajtoch committed:
fix: add `useEuiTheme` back to create_case_flyout.tsx

elasticmachine committed:
Merge branch 'main' into feat/eui-v111.0.0

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'feat/eui-v111.0.0' into kbn-flyout-system/example-iii

tsullivan committed:
temp build cleanup

tsullivan committed:
temp build cleanup

kibanamachine committed:
Changes from node scripts/capture_oas_snapshot   --include-path /api/status   --include-path /api/alerting/rule/   --include-path /api/alerting/rules   --include-path /api/actions   --include-path /api/security/role   --include-path /api/spaces   --include-path /api/streams   --include-path /api/fleet   --include-path /api/saved_objects/_import   --include-path /api/saved_objects/_export   --include-path /api/maintenance_window   --include-path /api/agent_builder --update

kibanamachine committed:
Changes from node scripts/eslint_all_files --no-cache --fix

tsullivan committed:
Merge branch 'kbn-flyout-system/example-iii' of github.com:tsullivan/kibana into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/378090)
* Commit: 9ecf07650592da776836d1dfd724fa60282c9c41

### Failed CI Steps
* [FTR Configs #2](https://buildkite.com/elastic/kibana-pull-request/builds/378090#019b901e-1010-41a0-851f-f2d58145232b)
* [FTR Configs #9](https://buildkite.com/elastic/kibana-pull-request/builds/378090#019b901e-1017-4ea3-85d3-7ef35a025dcf)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/378090#019b901e-1010-41a0-851f-f2d58145232b) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/378090/jobs/019b901e-1010-41a0-851f-f2d58145232b/artifacts/019b9040-34d8-4b8d-bd58-61d5760483c0) FTR Configs #<span></span>2 / Controls Dashboard control group hierarchical chaining "before all" hook for "Shows all available options in first Options List control"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/378090#019b901e-1017-4ea3-85d3-7ef35a025dcf) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/378090/jobs/019b901e-1017-4ea3-85d3-7ef35a025dcf/artifacts/019b9049-801b-459d-8ca5-16d46e2e56df) FTR Configs #<span></span>9 / Endpoint plugin @<span></span>ess @<span></span>serverless @<span></span>skipInServerlessMKI test metadata apis list endpoints GET route "before all" hook for "should return one entry for each host with default paging"

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/9ecf07650592da776836d1dfd724fa60282c9c41) | diff |
| --- | --- | --- | --- |
| `core` | 520 | 522 | +2 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/9ecf07650592da776836d1dfd724fa60282c9c41) | diff |
| --- | --- | --- | --- |
| `@kbn/core-overlays-browser` | 28 | 31 | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/9ecf07650592da776836d1dfd724fa60282c9c41) | diff |
| --- | --- | --- | --- |
| `agentBuilder` | 528.9KB | 528.9KB | +16.0B |
| `automaticImport` | 1.3MB | 1.3MB | +16.0B |
| `automaticImportVTwo` | 15.9KB | 15.9KB | +16.0B |
| `canvas` | 1.0MB | 1.0MB | +16.0B |
| `contentConnectors` | 467.1KB | 467.1KB | +16.0B |
| `dashboard` | 721.6KB | 721.6KB | +16.0B |
| `dataConnectors` | 16.2KB | 16.2KB | +16.0B |
| `enterpriseSearch` | 953.9KB | 953.9KB | +16.0B |
| `eventAnnotationListing` | 206.8KB | 206.9KB | +16.0B |
| `filesManagement` | 102.8KB | 102.8KB | +16.0B |
| `graph` | 372.5KB | 372.5KB | +16.0B |
| `home` | 66.9KB | 67.0KB | +16.0B |
| `indexManagement` | 734.0KB | 734.0KB | +16.0B |
| `infra` | 1.1MB | 1.1MB | +16.0B |
| `kibanaOverview` | 47.9KB | 47.9KB | +16.0B |
| `management` | 33.4KB | 33.4KB | +16.0B |
| `maps` | 3.1MB | 3.1MB | +16.0B |
| `ml` | 5.6MB | 5.6MB | +16.0B |
| `observabilityShared` | 36.4KB | 36.5KB | +16.0B |
| `osquery` | 1.1MB | 1.1MB | +16.0B |
| `searchGettingStarted` | 155.6KB | 155.6KB | +16.0B |
| `searchHomepage` | 184.4KB | 184.4KB | +16.0B |
| `searchIndices` | 215.9KB | 215.9KB | +16.0B |
| `searchInferenceEndpoints` | 142.4KB | 142.5KB | +16.0B |
| `searchPlayground` | 244.1KB | 244.1KB | +16.0B |
| `searchQueryRules` | 130.2KB | 130.3KB | +16.0B |
| `searchSynonyms` | 64.8KB | 64.8KB | +16.0B |
| `security` | 488.4KB | 488.4KB | +16.0B |
| `securitySolution` | 10.7MB | 10.7MB | +16.0B |
| `securitySolutionEss` | 50.1KB | 50.1KB | +16.0B |
| `securitySolutionServerless` | 72.2KB | 72.2KB | +16.0B |
| `spaces` | 215.4KB | 215.4KB | +16.0B |
| `visualizations` | 344.2KB | 344.2KB | +16.0B |
| `workplaceAIApp` | 94.8KB | 94.9KB | +16.0B |
| total |  |  | +544.0B |

#### Public APIs missing exports
> Total count of every type that is part of your API that should be exported but is not. This will cause broken links in the API documentation system. Target amount is 0. `Run node scripts/build_api_docs --plugin [yourplugin] --stats exports` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/9ecf07650592da776836d1dfd724fa60282c9c41) | diff |
| --- | --- | --- | --- |
| `@kbn/core` | 918 | 919 | +1 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/9ecf07650592da776836d1dfd724fa60282c9c41) | diff |
| --- | --- | --- | --- |
| `core` | 496.9KB | 498.6KB | +1.7KB |
| `esUiShared` | 96.3KB | 96.3KB | +16.0B |
| `kbnUiSharedDeps-srcJs` | 4.3MB | 4.3MB | +16.0B |
| total |  |  | +1.7KB |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/9ecf07650592da776836d1dfd724fa60282c9c41) | diff |
| --- | --- | --- | --- |
| `@kbn/core-overlays-browser` | 53 | 61 | +8 |


</details>


### History
* :green_heart: [Build #371894](https://buildkite.com/elastic/kibana-pull-request/builds/371894) succeeded 0ca280ac49f02e26d390588a416f03fb326a5c75
* :broken_heart: [Build #371862](https://buildkite.com/elastic/kibana-pull-request/builds/371862) failed 1d340040f1e551485d22e861e1c8ff1ce741d3cb
* :broken_heart: [Build #371460](https://buildkite.com/elastic/kibana-pull-request/builds/371460) failed 396128424656a125db1029930ca6cb44e482598a
* :broken_heart: [Build #369184](https://buildkite.com/elastic/kibana-pull-request/builds/369184) failed dd620ec65478b9bea788dddcf106aa7acc0c0010
* :broken_heart: [Build #369153](https://buildkite.com/elastic/kibana-pull-request/builds/369153) failed f0112582a3b8c20f0131d56a9c34a88f58d38afa
* :broken_heart: [Build #366100](https://buildkite.com/elastic/kibana-pull-request/builds/366100) failed f681e3f5adaafc792b2d84f57369a9ed800e49ed

cc @tsullivan

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378090","number":378090,"commit":"9ecf07650592da776836d1dfd724fa60282c9c41"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371894","number":371894,"commit":"0ca280ac49f02e26d390588a416f03fb326a5c75"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371862","number":371862,"commit":"1d340040f1e551485d22e861e1c8ff1ce741d3cb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371460","number":371460,"commit":"396128424656a125db1029930ca6cb44e482598a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/369184","number":369184,"commit":"dd620ec65478b9bea788dddcf106aa7acc0c0010"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/369153","number":369153,"commit":"f0112582a3b8c20f0131d56a9c34a88f58d38afa"}],"number":378090}
buildkite-pr-comment-->

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'kbn-flyout-system/example-iii' of github.com:tsullivan/kibana into kbn-flyout-system/example-iii

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
clean up app menu and simplify

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

kowalczyk-krzysztof commented:
@tsullivan 
I'm testing the example plugin and overall it seems to be working as intended, except for when you already have a "flyout with core.overlays services" open and you try to toggle the push or focus toggles - they have no effect, they only apply when toggled before opening a flyout. This works fine for "Flyouts with EuiFlyout". Is this just a bug in the examples or some bug in the implementation of the service?

kowalczyk-krzysztof commented on src/core/packages/overlays/browser-internal/src/flyout/system_flyout_service.tsx:
```diff
@@ -0,0 +1,118 @@
+        content: React.ReactElement,
+        { session = 'start', title, ...options }: OverlaySystemFlyoutOpenOptions = {}
+      ): OverlayRef => {
+        const flyoutId = `system-flyout-${++this.idCounter}`;
```
Why not simply use a uuidv4? This would also allow us to get rid of `idCounter` which seems to only be used to get a new id.

kowalczyk-krzysztof commented on src/core/packages/overlays/browser/src/flyout.ts:
```diff
@@ -32,12 +32,19 @@ export interface OverlayFlyoutStart {
+  /**
+   * Use overlays.openSystemFlyout to create session-based flyouts.
+   */
+  session?: 'never';
```
Wouldn't native TypeScript `never` be better here?

kowalczyk-krzysztof reviewed (approved):
This approach seems fine and the example plugin works as expected. I left some nitpick questions, feel free to ignore them.

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
updates per feedback

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

TinaHeiligers reviewed (approved):
LGTM with a potential follow up

tsullivan committed:
Merge branch 'main' into kbn-flyout-system/example-iii

tsullivan committed:
update comments for intellisense

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/381114)
* Commit: 38c57e611c3ed82e8d7e12e55a7b893b50355103

### Failed CI Steps
* [FTR Configs #24](https://buildkite.com/elastic/kibana-pull-request/builds/381114#019bb8dc-7fb7-4906-8d8d-e5aa0a6bbce8)
* [FTR Configs #51](https://buildkite.com/elastic/kibana-pull-request/builds/381114#019bb8dc-7fd7-4011-aab4-e6d2b46b5a00)
* [FTR Configs #85](https://buildkite.com/elastic/kibana-pull-request/builds/381114#019bb8dc-8004-4905-88a9-16f9b37471f6)
* [FTR Configs #123](https://buildkite.com/elastic/kibana-pull-request/builds/381114#019bb8dc-8033-416f-984d-5bdebdedc932)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/381114#019bb8dc-7fd7-4011-aab4-e6d2b46b5a00) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/381114/jobs/019bb8dc-7fd7-4011-aab4-e6d2b46b5a00/artifacts/019bb906-b820-4954-a949-5eb374b68b6e) FTR Configs #<span></span>51 / Endpoint plugin @<span></span>ess @<span></span>serverless @<span></span>skipInServerlessMKI test metadata apis list endpoints GET route "before all" hook for "should return one entry for each host with default paging"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/381114#019bb8dc-7fb7-4906-8d8d-e5aa0a6bbce8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/381114/jobs/019bb8dc-7fb7-4906-8d8d-e5aa0a6bbce8/artifacts/019bb905-33aa-44f1-9174-04b04d6324d3) FTR Configs #<span></span>24 / Endpoint plugin spaces support @<span></span>ess @<span></span>serverless @<span></span>skipInServerlessMKI Endpoint management space awareness support "before all" hook in "@<span></span>ess @<span></span>serverless @<span></span>skipInServerlessMKI Endpoint management space awareness support"

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/481d74df83bc63fd536f39016313f1304a191a99) | [after](https://github.com/elastic/kibana/commit/38c57e611c3ed82e8d7e12e55a7b893b50355103) | diff |
| --- | --- | --- | --- |
| `core` | 542 | 544 | +2 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/481d74df83bc63fd536f39016313f1304a191a99) | [after](https://github.com/elastic/kibana/commit/38c57e611c3ed82e8d7e12e55a7b893b50355103) | diff |
| --- | --- | --- | --- |
| `@kbn/core-overlays-browser` | 28 | 31 | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/481d74df83bc63fd536f39016313f1304a191a99) | [after](https://github.com/elastic/kibana/commit/38c57e611c3ed82e8d7e12e55a7b893b50355103) | diff |
| --- | --- | --- | --- |
| `agentBuilder` | 532.2KB | 532.2KB | +16.0B |
| `automaticImport` | 1.3MB | 1.3MB | +16.0B |
| `automaticImportVTwo` | 38.4KB | 38.4KB | +16.0B |
| `canvas` | 1.0MB | 1.0MB | +16.0B |
| `contentConnectors` | 467.1KB | 467.1KB | +16.0B |
| `dashboard` | 786.5KB | 786.5KB | +16.0B |
| `dataConnectors` | 67.8KB | 67.8KB | +16.0B |
| `enterpriseSearch` | 954.3KB | 954.3KB | +16.0B |
| `eventAnnotationListing` | 207.0KB | 207.0KB | +16.0B |
| `filesManagement` | 103.0KB | 103.0KB | +16.0B |
| `graph` | 372.6KB | 372.7KB | +16.0B |
| `home` | 66.9KB | 67.0KB | +16.0B |
| `indexManagement` | 734.3KB | 734.3KB | +16.0B |
| `infra` | 1.2MB | 1.2MB | +16.0B |
| `kibanaOverview` | 47.9KB | 47.9KB | +16.0B |
| `management` | 33.4KB | 33.4KB | +16.0B |
| `maps` | 3.1MB | 3.1MB | +16.0B |
| `ml` | 5.6MB | 5.6MB | +16.0B |
| `observabilityShared` | 36.4KB | 36.5KB | +16.0B |
| `osquery` | 1.1MB | 1.1MB | +16.0B |
| `searchGettingStarted` | 155.9KB | 156.0KB | +16.0B |
| `searchHomepage` | 185.5KB | 185.5KB | +16.0B |
| `searchIndices` | 215.7KB | 215.7KB | +16.0B |
| `searchInferenceEndpoints` | 142.4KB | 142.5KB | +16.0B |
| `searchPlayground` | 244.1KB | 244.1KB | +16.0B |
| `searchQueryRules` | 130.2KB | 130.3KB | +16.0B |
| `searchSynonyms` | 64.8KB | 64.8KB | +16.0B |
| `security` | 491.3KB | 491.3KB | +16.0B |
| `securitySolution` | 10.8MB | 10.8MB | +16.0B |
| `securitySolutionEss` | 49.1KB | 49.2KB | +16.0B |
| `securitySolutionServerless` | 71.3KB | 71.3KB | +16.0B |
| `spaces` | 215.4KB | 215.4KB | +16.0B |
| `visualizations` | 344.8KB | 344.8KB | +16.0B |
| `workplaceAIApp` | 95.3KB | 95.3KB | +16.0B |
| total |  |  | +544.0B |

#### Public APIs missing exports
> Total count of every type that is part of your API that should be exported but is not. This will cause broken links in the API documentation system. Target amount is 0. `Run node scripts/build_api_docs --plugin [yourplugin] --stats exports` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/481d74df83bc63fd536f39016313f1304a191a99) | [after](https://github.com/elastic/kibana/commit/38c57e611c3ed82e8d7e12e55a7b893b50355103) | diff |
| --- | --- | --- | --- |
| `@kbn/core` | 932 | 933 | +1 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/481d74df83bc63fd536f39016313f1304a191a99) | [after](https://github.com/elastic/kibana/commit/38c57e611c3ed82e8d7e12e55a7b893b50355103) | diff |
| --- | --- | --- | --- |
| `core` | 500.2KB | 501.9KB | +1.6KB |
| `esUiShared` | 96.3KB | 96.3KB | +16.0B |
| `kbnUiSharedDeps-srcJs` | 4.4MB | 4.4MB | +16.0B |
| total |  |  | +1.7KB |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/481d74df83bc63fd536f39016313f1304a191a99) | [after](https://github.com/elastic/kibana/commit/38c57e611c3ed82e8d7e12e55a7b893b50355103) | diff |
| --- | --- | --- | --- |
| `@kbn/core-overlays-browser` | 53 | 61 | +8 |


</details>


### History
* :broken_heart: [Build #380618](https://buildkite.com/elastic/kibana-pull-request/builds/380618) failed 8bf9b580cd951bbff3622bef1b72d4e8f8e41176
* :broken_heart: [Build #379709](https://buildkite.com/elastic/kibana-pull-request/builds/379709) failed 083fb05704ac30d6d112aa85616590e9b0f7f51b
* :broken_heart: [Build #379318](https://buildkite.com/elastic/kibana-pull-request/builds/379318) failed 0eb5ae99bd6a710093e3d7b21c63133e915fd716
* :broken_heart: [Build #379188](https://buildkite.com/elastic/kibana-pull-request/builds/379188) failed 6e99a9773355f083b0afafb70187b0dda9287f75
* :broken_heart: [Build #378648](https://buildkite.com/elastic/kibana-pull-request/builds/378648) failed defb602047fcd83ce1a88f1f54425be43ef891df

cc @tsullivan

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/381114","number":381114,"commit":"38c57e611c3ed82e8d7e12e55a7b893b50355103"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/380618","number":380618,"commit":"8bf9b580cd951bbff3622bef1b72d4e8f8e41176"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/379709","number":379709,"commit":"083fb05704ac30d6d112aa85616590e9b0f7f51b"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/379318","number":379318,"commit":"0eb5ae99bd6a710093e3d7b21c63133e915fd716"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/379188","number":379188,"commit":"6e99a9773355f083b0afafb70187b0dda9287f75"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378648","number":378648,"commit":"defb602047fcd83ce1a88f1f54425be43ef891df"}],"number":381114}
buildkite-pr-comment-->
